### PR TITLE
Task#13 - FEAT: Create footer section

### LINKED ISSUE #14: DevTask - Create section components
- [x] #15 
- [x] #17 
- [x] #19
- [x] #21 

After fix reponsive for all section
- [x] #32

### LINKED ISSUE #32: Task#13 - DevTask: Create footer section
## OVERVIEW
- Create footer section
- Update responsive for footer section

## DEMO

image

image

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Closes #32 

## OVERVIEW
- Create component FooterCard, FooterMenu,FooterMenuItem
- Install icon img
- Create footer section
- Updated reponsive 

## DEMO 

https://github.com/Agility-Internship/Duong-The-Hien-ReactJS/assets/113186169/8ea67d62-b0f2-4756-8945-dc22b3f8ea9f

### CODE REVIEW

File: practice-one/src/Components/Section/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                <img
+                  src="src\assets\Linkedin.png"
+                  alt="Here is a logo of the media above"
+                />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </FooterCard>
+      <FooterMenu title="Company">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: why don't use `map`. Please keep in mind that if it's react and we have a list, we should use `map` to render into the UI. Do not repeat this chunk code

File: practice-one/src/Components/Section/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    <hr className="divider mx-8 h-px bg-lucky-grey opacity-24" />
+
+    <div className="footer-rule container mx-auto pt-8">
+      <div className="nav-menu flex flex-col lg:flex-row gap-6 lg:gap-0 justify-between">
+        <h3 className="item">
+          <a className="text-gray-500" href="__blank">
+            &copy;Copyright 2022 Yoora, Inc.
+          </a>
+        </h3>
+        <ul className="menu-list flex flex-col lg:flex-row lg:gap-6">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: the same here, it's a list

File: practice-one/src/Components/Section/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    </a>
+  </div>
+);
+
+FooterLogo.propTypes = {
+  logoSrc: PropTypes.string.isRequired,
+  altText: PropTypes.string.isRequired,
+};
+
+const FooterCard = ({ children }) => <div className="card w-[30%] lg:w-[20%]">{children}</div>;
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: Do we really need this component

File: practice-one/src/Components/Section/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+const Footer = () => (
+  <footer className="footer bg-white relative mt-56 lg:mt-0 m-7">
+    <div className="footer-content container mx-auto mt-14 mb-12 flex flex-col lg:flex-row flex-wrap justify-between gap-y-8 lg:flex-nowrap">
+      <FooterCard>
+        <FooterLogo logoSrc="src\assets\logo.png" altText="logo" />
+        <FooterCardBody text="We built an elegant solution. Our team created a fully integrated sales and marketing solution for SMBs" />
+        <div className="card-footer">
+          <ul className="media menu-list flex gap-5">
+            <li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: why dont we reuse it 
image

File: practice-one/src/App.jsx [hunk 5]
```diff
@@ -5,13 +5,15 @@ import Navbar from './Components/Section/Navbar';
 import Header from './Components/Section/Header';
 import ListCard from './Components/Section/ProductList';
 import CtaSection from './Components/Section/CTA';
+import Footer from './Components/Section/Footer';
 
 const App = () => (
   <div>
```
Comment (phattruong-agilityio) [inline practice-one/src/App.jsx]: do we need this div

File: practice-one/src/Components/Section/Footer.jsx (hunk 1 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Yes,I missed them, so I updated the lists and used map to render into UI in the latest commit

File: practice-one/src/Components/Section/Footer.jsx (hunk 2 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Yes, I updated

File: practice-one/src/Components/Section/Footer.jsx (hunk 3 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Yes, It's useless so i removed this

File: practice-one/src/Components/Section/Footer.jsx (hunk 4 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Yes, resues FooterMenuItem component and using map to render list

File: practice-one/src/App.jsx (hunk 5 above)
Comment (TheHienDuong) [reply practice-one/src/App.jsx]: it's not necessary, just use <></>.
Thanks for reminding

File: practice-one/src/Components/Section/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+            <img
+              src="src\assets\Shape.png"
+              alt="This is the icon for the change language button"
+            />
+            <select
+              className="btn dropdown-toggle font-medium text-base"
+              name="language"
+              id="language"
+            >
+              <option value="volvo">English</option>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: check the value

File: practice-one/src/Components/Section/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+              <option value="saab">Vietnamese</option>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: check the value

File: practice-one/src/Components/Section/Footer.jsx (hunk 6 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Done

File: practice-one/src/Components/Section/Footer.jsx (hunk 7 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Done

**Pull request merged**